### PR TITLE
Fix: update readme after new tagging mechanic

### PR DESCRIPTION
**Pull request by Artyop**
Following the pr #286, update of the README.md about how the tagging mechanic works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,14 +62,12 @@ Here is how testing is accomplished in the `llvm` image:
 
 ### Making changes
 
-All images get automatic tags based on checked-in contents of image subdirectory. At any point in git history of a subdirectory
-there exists a unique [git tree object hash](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects), that is what's used for
-image tags.
+All images get automatic tags based on the latest commit of the image directory. Tags used to be done through unique [git tree object hash](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects) but due to setting up renovate to automatically update images, and the inability to order sha256 value by time, the new tagging system use a combination of the timestamp and the short sha1 of the latest directory's image commit, using the [git log command with the image directory as a path parameter](https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History).
```
Comment (qmonnet) [inline README.md:65]: It's sha1 and not sha256, isn't it?

```suggestion
All images get automatic tags based on the latest commit of the image directory. Tags used to be based on unique [git tree object hash](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects), but due to setting up renovate to automatically update images, and the inability to order sha1 values by time, the new tagging system uses a combination of the timestamp and the short sha1 of the latest commit for the image directory, using the [git log command with the image directory as a path parameter](https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History).
```

File: README.md [hunk 2]
```diff
@@ -62,14 +62,12 @@ Here is how testing is accomplished in the `llvm` image:
...
 
 As the result of this, following stands:
 
 - image build definitions can be obtained with `git show <tag>`
-- image build is defined by contents of a directory
+- image build is defined by the last commit that has modified the image directory
```
Comment (qmonnet) [inline README.md:70]:
```suggestion
- image build is defined by the last commit that modified the image directory
```

**Review by qmonnet — CHANGES_REQUESTED** [review]
Small nits in the wording, but looks good otherwise, thanks!

**Comment by Artyop** [comment]
> Small nits in the wording, but looks good otherwise, thanks!

thank you, applied the fix !

**Review by qmonnet — APPROVED** [review]
Thank you! For future PRs, please squash the two commits together (do keep logical commits in the PR, but review fixes should not stand out as separate commits), and no need to keep the `co-authored-by` for trivial review fixes :slightly_smiling_face:

**Comment by Artyop** [comment]
> Thank you! For future PRs, please squash the two commits together (do keep logical commits in the PR, but review fixes should not stand out as separate commits), and no need to keep the `co-authored-by` for trivial review fixes 🙂

fixed that too, thanks a lot for the review !

**Comment by qmonnet** [comment]
I can't merge without the missing reviews. I'll request one from Tobias, as I think Daniel is out.

**Review by tklauser — APPROVED** [review]

**Pull request merged**